agent/assetmanager/manager.go

```diff
@@ -77,15 +77,16 @@ func getSystemEnviron() []string {
 
 	for _, e := range env {
 		pair := strings.Split(e, "=")
-		key, _ := pair[0], pair[1]
+		// Transform the key to uppercase because Windows uses "Path" and not "PATH"
+		key := strings.ToUpper(pair[0])
 
-		if presentVars[key] != true {
+		if !presentVars[key] {
 			presentVars[key] = true
```

Comment:
Looks like we can unconditionally set presentVars[key] to true here.

Reply:
Ha, good catch!